Allow the compiler, compiler flags, and libraries provided by the build environment to take precedence.
Author: Emik Lin (HKUMed CPOS)
--- lumpy-sv-0.3.1.orig/Makefile	2019-04-17 01:13:49.000000000 +0800
+++ lumpy-sv-0.3.1/Makefile	2026-08-13 00:00:00.000000000 +0800
@@ -19 +19 @@
-export CXX		= g++
+export CXX		?= g++
@@ -25,2 +25,2 @@
-export CXXFLAGS = -Wall -O2 -D_FILE_OFFSET_BITS=64 -fPIC 
-export LIBS		= -lz -L $(ZLIB_PATH)
+export CXXFLAGS ?= -Wall -O2 -D_FILE_OFFSET_BITS=64 -fPIC 
+export LIBS		?= -lz -L $(ZLIB_PATH)
